src/main/java/net/openhft/chronicle/threads/MediumEventLoop.java

```diff
@@ -180,7 +180,7 @@ private void stopEventLoopThread() {
     public void addHandler(@NotNull final EventHandler handler) {
         throwExceptionIfClosed();
 
-        checkInterrupted();
+        checkInterruptedAddingNewHandler();
```

Comment:
Do we really need to check this here? seems like the value is in breaking out of that blocking loop below.

Reply:
IMHO, Remove this check.